fe/fe-core/src/main/java/com/starrocks/sql/analyzer/WorkGroupAnalyzer.java

```diff
@@ -153,7 +153,7 @@ public static void analyzeProperties(WorkGroup workgroup, Map<String, String> pr
 
             if (key.equalsIgnoreCase(WorkGroup.CONCURRENCY_LIMIT)) {
                 int concurrencyLimit = Integer.parseInt(value);
-                if (concurrencyLimit <= 0) {
+                if (concurrencyLimit < 0) {
```

Comment:
why remove equal ?

Reply:
`concurrenctLimit == 0` should means disable this limitation.